src/Microsoft.Health.SqlServer/Features/Schema/Model/ColumnUtilities.cs

```diff
@@ -0,0 +1,28 @@
+﻿// -------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+// -------------------------------------------------------------------------------------------------
+
+using System;
+
+namespace Microsoft.Health.SqlServer.Features.Schema.Model
+{
+    internal static class ColumnUtilities
+    {
+        internal static long GetLengthForFloatColumn(byte precision)
+        {
+            if (precision >= 1 && precision <= 23)
```

Comment:
In the docs, it looks like the upper bound is 24, not 23? https://docs.microsoft.com/en-us/sql/t-sql/data-types/float-and-real-transact-sql?view=sql-server-ver15#syntax